### ISSUE TITLE
QR code parse error, error = R: No MultiFormat Readers were able to detect the code.

### ISSUE THREAD

**Issue by Asad9299** [mebjas/html5-qrcode#488]
Hello there,

I am using this plugin for scanning QR Codes from last one month, everything was working fine but one QR gives me the above mentioned error while scanning. I have checked that QR with the demo link (https://blog.minhazav.dev/research/html5-qrcode.html#scan-using-file) as well and there also it gives me the same mentioned error. Here, I am attaching that QR image for your reference. Please look into it and provide its solution as earliest.

https://user-images.githubusercontent.com/88971111/174524621-bdcd7ca8-04ea-4a08-836f-1b5fbb733578.png

 Thanks, 
 Asad Mansuri.

**Comment by Asad9299** [comment]
Yes, tried with different QR library and it got scanned easily. I have tried using https://nimiq.github.io/qr-scanner/demo/ and it got scanned successfully.

**Comment by ROBERT-MCDOWELL** [comment]
what are your object configuration?

**Comment by Asad9299** [comment]
I have used the https://github.com/SimpleSoftwareIO/simple-qrcode for generating QR code in laravel. The size of QR was 500 * 500px.

**Comment by ROBERT-MCDOWELL** [comment]
I need you share your code please.

**Comment by Asad9299** [comment]
I have written below code for scanning QR.

 <script src="https://unpkg.com/html5-qrcode@2.0.9/dist/html5-qrcode.min.js"></script>
 
 <script type="text/javascript" async>
        function onScanSuccess(decodedText, decodedResult) {
          html5QrcodeScanner.clear();
        }
        
         var html5QrcodeScanner = new Html5QrcodeScanner(
            "qr-reader", {
                fps: 10,
                qrbox: {width: 500, height: 500}
            });
        html5QrcodeScanner.render(onScanSuccess);
 </script>
 
 Please provide some solution for the mentioned error.

**Comment by Asad9299** [comment]
Yes, I have tried around 100+ QRs, except the shared above all qrs were successfully scanned. Can you please check why that particular QR has an error while scanning?

**Comment by ROBERT-MCDOWELL** [comment]
it's your job to check why some qr code don't pass, it's not a html5-qrcode problem.

**Comment by Asad9299** [comment]
No change, same issue persisted.

**Comment by ROBERT-MCDOWELL** [comment]
this is not my demo link, I'm a contributor/user, this is not my project.
you should share OS, browser, version

**Comment by Asad9299** [comment]
OS: Ubuntu 18.04
Browser: Google Chrome
HTML5-Scanner Version: 2.0.9

**Comment by ROBERT-MCDOWELL** [comment]
ok first of all , update html5-qrcode to the latest version (2.2.1) or git.
then try again and report the results.

**Comment by Asad9299** [comment]
Could you please let me know from where I can get that? Since, I am using the version given in demo link.

**Comment by ROBERT-MCDOWELL** [comment]
https://github.com/mebjas/html5-qrcode/releases/tag/v2.2.1

**Comment by Asad9299** [comment]
used this latest js but still no luck! Issue persisted again.

**Comment by ROBERT-MCDOWELL** [comment]
try several qr-code that used to work for you and tell me if it's ok.

**Comment by ROBERT-MCDOWELL** [comment]
I tried this Qrcode on my side and indeed it does not work. I suspect it's a corrupted one.
you said it was working one month before but before what? chrome update? other?
did you try this qrcode with other scanners? if yes and works, could you see any result?

**Comment by ROBERT-MCDOWELL** [comment]
I uploaded this qrcode at https://qrscanneronline.com/
and the result is
1ecddb0d-72ae-662c-8091-8cec4bc712b8
is it the result you are expecting for?

**Comment by ROBERT-MCDOWELL** [comment]
Ok I think I understand what's happening. your Qrcode image does not have white margins and confuse the algorithm.
try to add white margins all arround the qrcode and see if it works. if yes, I suggest to open a "feature" issue asking to add white margins on uploaded images or you can do it yourself and create a PR.

**Comment by Asad9299** [comment]
Yes, the result is correct which you have shared. I have tried with black margins but then sometimes it got scanned with black margin and some time it gives the same error.

**Comment by Asad9299** [comment]
What's the conclusion? Should I continue using *Scan with Camera* option instead of *Scan with Image* till the time some solution is provided? Your thoughts?

**Comment by ROBERT-MCDOWELL** [comment]
I suggest you search the issue in the code and create a PR to fix it for the project.
